src/main/java/gov/cms/madie/models/measure/TestCaseSequence.java

```diff
@@ -0,0 +1,18 @@
+package gov.cms.madie.models.measure;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@Document
+public class TestCaseSequence {
+  @Id private String id;
```

Comment:
is the measure id going to be the sequence id?

Reply:
yeah this will be mapped with the measure Id